test/runtests.jl

```diff
@@ -102,6 +102,12 @@ facts("Tests for Taylor1 expansions") do
     @fact asin(t) + acos(t) == pi/2 --> true
     @fact derivative(acos(t)) == - 1/sqrt(1-t^2) --> true
 
+    @fact - sinh(t) + cosh(t) == exp(-t) --> true
+    @fact  sinh(t) + cosh(t) == exp(t) --> true
+    @fact evaluate(- sinh(t)^2 + cosh(t)^2 , rand()) == 1 --> true
+    @fact tanh(t + 0im) == -1im * tan(t*1im) --> true
+    @fact  evaluate(tanh(t/2),1.5) == evaluate(sinh(t) / (cosh(t) + 1),1.5) --> true
+    
```

Comment:
It's worth to add tests that involve identities with complex numbers and other functions. For instance,
```julia
@fact cosh(t) == real(cos(im*t)) --> true
@fact sinh(t) == imag(sin(im*t)) --> true
```